layouts/partials/pages/home-posts.html

```diff
@@ -13,9 +13,7 @@ <h2 class="post-title">
                     {{ partial "utils/summary.html" . }}
                 </summary>
                 {{ if .Truncated }}
```

Comment:
Actually, we can solve this issue simply by adding a `or` condition, `{{ if or .Truncated .Params.summary }}`.

Reply:
Yes, that seems to be the better solution.